src/Compilers/CSharp/Portable/FlowAnalysis/AbstractFlowPass.cs

```diff
@@ -2051,7 +2051,7 @@ public override BoundNode VisitPropertyAccess(BoundPropertyAccess node)
         {
             var property = node.PropertySymbol;
 
-            if (Binder.AccessingAutoPropertyFromConstructor(node, _symbol))
+            if (Binder.IsPropertyAssignableFromConstructor(node, _symbol)) // PROTOTYPE(semi-auto-props): Revise this method call is the behavior we want and add unit tests..
```

Comment:
>IsPropertyAssignableFromConstructor [](http://example.com/codeflow?start=23&length=35)

Similar comment here as well. I think we are specifically dealing with read access here.